stall.rb

```diff
@@ -0,0 +1,24 @@
+require 'car'
+
+class Stall
+  attr_reader :level, :stall_num, :type, :filled, :car
+
+  def initialize(args)
```

Comment:
Conventionally, since it is just ONE actual arg, we call this an options hash, and receive it as `opts`. :]